Configuration/TCA/Module/Options.php

```diff
@@ -121,7 +127,7 @@
             'directories,threshold,file_types,
 			--palette--;LLL:EXT:frontend/Resources/Private/Language/locallang_ttc.xlf:ALT.imgDimensions;dimensions,
 			--div--;LLL:EXT:image_autoresize/Resources/Private/Language/locallang_tca.xlf:tabs.options,
-				auto_orient,keep_metadata,resize_png_with_alpha,conversion_mapping,
+				auto_orient,auto_orient_custom,keep_metadata,resize_png_with_alpha,conversion_mapping,
```

Comment:
Since this is fully related to `auto_orient`, I'd suggest to group both "auto autorientation" options in a single palette instead